### PR TITLE
epp to copy udfs

### PR DESCRIPTION
**Pull request by mayabrandi**
This PR adds a EPP to copy udfs from art to samp.  It also restructures the project and cli a little 

**Test new script:**
- [x] lims epps udf copy artifact-to-sample

**Test restructured cli:**
- [x] lims epps files make-kapa-csv
- [x] lims epps move move-samples 
- [x] lims epps move place-samples-in-seq-agg 
- [x] lims epps move rerun-samples  

**When the deploying on production, the configuration of the scripts need to be updated.**

### Review:
- [ ] Code approved by
- [x] Tests executed by @mayabrandi 
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [x] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: cg_lims/EPPs/get_and_set/art2samp.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+from cg_lims.exceptions import LimsError, MissingUDFsError
```
Comment (mayabrandi) [inline cg_lims/EPPs/get_and_set/art2samp.py:2]: Is this dir really supposed to be named get_and_set? I cant figure out. @Mropat

**Comment by mayabrandi** [comment]
**Testing the new epp artifact-to-sample**
Step:
Skärmavbild 2020-10-14 kl  09 28 10
Running script from commandline:
Skärmavbild 2020-10-14 kl  09 27 54
Output as expected.
Skärmavbild 2020-10-14 kl  09 26 30
udf set on sample.

Conclusion. New script ready for production

**Comment by mayabrandi** [comment]
**Testing the new cli**

Running the command
Skärmavbild 2020-10-14 kl  09 42 21
I double checked that samples were moved 👍

**Review by Mropat — APPROVED** [review]
Very nice! However, I suggest to add __pycache__ and other stuff unrelated to project to .gitignore file!

**Pull request merged**